cellfinder_napari/tests/test_curation.py

```diff
@@ -1,11 +1,32 @@
+import pytest
+from napari.layers import Points
+
 from cellfinder_napari import sample_data
 from cellfinder_napari.curation import CurationWidget
 
 
-def test_detect_function(make_napari_viewer):
+@pytest.fixture
+def curation_widget(make_napari_viewer):
     """
-    Smoke test that adding the curation widget to napari works
+    Create a viewer, add the curation widget, and return the widget.
+    The viewer can be accessed using ``widget.viewer``.
     """
     viewer = make_napari_viewer()
     widget = CurationWidget(viewer)
     viewer.window.add_dock_widget(widget)
+    return widget
+
+
+def test_add_new_training_layers(curation_widget):
+    viewer = curation_widget.viewer
+    layers = viewer.layers
+    # Check that layers list starts off empty
+    assert len(layers) == 0
+    curation_widget.add_training_data()
+    assert len(layers) == 2
+
+    for layer in layers:
+        assert isinstance(layer, Points)
```

Comment:
```suggestion
    assert all(isinstance(layer, Points) for layer in layers)
```

Reply:
Ah that's much nicer 👍

Reply:
Probably faster too (granted it is only two entries 😁)